packages/swapper/src/swappers/ArbitrumBridgeSwapper/endpoints.ts

```diff
@@ -114,6 +114,10 @@ export const arbitrumBridgeApi: SwapperApi = {
     const { buyAsset, sellAsset, sellAmountIncludingProtocolFeesCryptoBaseUnit } = step
     const { receiveAddress } = tradeQuote
 
+    // TODO(gomes): when we actually split between TradeQuote and TradeRate in https://github.com/shapeshift/web/issues/7941,
+    // this won't be an issue anymore
```

Comment:
i.e not for this PR - this is intended to be a minimal rate *exposing* quotes without BIP44 assuming flag on, so runtime guards are added for paranoia